From 267bea1d773edaa0cf15ac04a5cee8a1683c99cd Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 27 Apr 2018 20:24:06 +0200
Subject: [PATCH 0599/1051] arm64: dts: marvell: mcbin: add comphy references
 to USB port

Adding phandle to the USB port will allow HCD driver to configure
appropriate marvell, comphy-cp110 accordingly.

Change-Id: I838a8e2c504192ce929cbf0f8f2a2348460f3e35
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index cad9d1b1c1c4..3a0c55320da6 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -396,4 +396,6 @@
 	/* CPS Lane 2 - CON7 */
 	usb-phy = <&usb3h0_phy>;
 	status = "okay";
+	/* Generic PHY, providing serdes lanes */
+	phys = <&cp1_comphy2 0>;
 };
-- 
2.17.1

